okhttp/src/main/java/okhttp3/internal/Platform.java

```diff
@@ -176,6 +193,13 @@ private static Platform findPlatform() {
     return new Platform();
   }
 
+  /**
+   * Is Java 9.  Does not include Java 10 or later.
```

Comment:
Typically this isn’t how we detect the host platform. Instead it’s generally more reliable to look for signatures like the presence of methods like `setApplicationProtocols()`. Otherwise we regress on Java 10 for example.